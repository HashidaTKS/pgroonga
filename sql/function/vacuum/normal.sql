CREATE TABLE memos (
  content text
);

CREATE INDEX pgroonga_index ON memos USING PGroonga (content);

SELECT pgroonga_table_name('pgroonga_index');
\gset old_
REINDEX INDEX pgroonga_index;

SELECT pgroonga_command('select',
                        ARRAY[
                          'table', :'old_pgroonga_table_name'
                        ])::json#>>'{0, 0}';
SELECT pgroonga_vacuum();
SELECT pgroonga_command('select',
                        ARRAY[
                          'table', :'old_pgroonga_table_name'
                        ])::json#>>'{0, 0}';

DROP TABLE memos;
