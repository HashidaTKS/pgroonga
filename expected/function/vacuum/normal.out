CREATE TABLE memos (
  content text
);
CREATE INDEX pgroonga_index ON memos USING PGroonga (content);
SELECT pgroonga_table_name('pgroonga_index');
 pgroonga_table_name 
---------------------
 Sources21116
(1 row)

\gset old_
REINDEX INDEX pgroonga_index;
SELECT pgroonga_command('select',
                        ARRAY[
                          'table', :'old_pgroonga_table_name'
                        ])::json#>>'{0, 0}';
 ?column? 
----------
 0
(1 row)

SELECT pgroonga_vacuum();
 pgroonga_vacuum 
-----------------
 t
(1 row)

SELECT pgroonga_command('select',
                        ARRAY[
                          'table', :'old_pgroonga_table_name'
                        ])::json#>>'{0, 0}';
 ?column? 
----------
 -22
(1 row)

DROP TABLE memos;
